go/flags/endtoend/vttablet.txt

```diff
@@ -350,8 +350,10 @@ Usage of vttablet:
       --tx-throttler-default-priority int                                Default priority assigned to queries that lack priority information (default 100)
       --tx-throttler-dry-run                                             If present, the transaction throttler only records metrics about requests received and throttled, but does not actually throttle any requests.
       --tx-throttler-healthcheck-cells strings                           Synonym to -tx_throttler_healthcheck_cells
+      --tx-throttler-query-pool-thresholds int:int                       The low:high query pool usage percentages for the transaction throttler to begin throttling low-priority select queries, defined as two colon-separated integers between 0 and 100.
       --tx-throttler-tablet-types strings                                A comma-separated list of tablet types. Only tablets of this type are monitored for replication lag by the transaction throttler. Supported types are replica and/or rdonly. (default replica)
       --tx-throttler-topo-refresh-interval duration                      The rate that the transaction throttler will refresh the topology to find cells. (default 5m0s)
+      --tx-throttler-tx-pool-thresholds int:int                          The low:high tx pool usage percentages for the transaction throttler to begin throttling low-priority transactions, defined as two colon-separated integers between 0 and 100.
```

Comment:
can we have 1 configuration and use it for all the pools? Otherwise, we will end up having at least 4 flags